app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -289,6 +297,12 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
                     false
                 }
             }
+            R.id.historyFragment -> {
```

Comment:
Sounds good, I'll use the when format.  However, I won't be able to remove the `fromHomeFragment` since it is used as a parameter for couple function calls.  But I'll move it back to it's original position.  Thanks